public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Yes create react app framed in that way.. it will look for the index.html file inside the public folder. This directory created by default we just made changes in the file accordingly